tutorial/notebooks/README.md

```diff
@@ -34,7 +34,7 @@ $ jupyter-notebook --no-browser
 [C 10:56:46.529 NotebookApp]
 
     To access the notebook, open this file in a browser:
-        file:///home/aporter/.local/share/jupyter/runtime/nbserver-463-open.html
+        file:///home/me/.local/share/jupyter/runtime/nbserver-463-open.html
```

Comment:
I understand this link wont work. Should we delete this option altogether in favour of the localhost links below?

Reply:
Meh, no links will work because they rely on the unique id generated by Jupyter anyway.
I changed that because I was trying to get rid of as many appearances of your usernames and absolute paths in the repo as possible. Besides, if you notice, in the first two lines of this output, it'd seem @arporter had already made a conscious change from his username to this generic one, I just followed his idea. :)

Reply:
Ok, I am good with this since it is out of scope from the main purpose of this PR. I will try to make them working links in another branch that I have relating to the tutorials.

Reply:
Hmmm, yes, maybe there's an option to tell Jupyter to place the server on a designated link.